velox/connectors/hive/tests/PartitionIdGeneratorTest.cpp

```diff
@@ -33,7 +33,7 @@ class PartitionIdGeneratorTest : public ::testing::Test,
 TEST_F(PartitionIdGeneratorTest, consecutiveIdsSingleKey) {
   auto numPartitions = 100;
 
-  PartitionIdGenerator idGenerator(ROW({VARCHAR()}), {0}, 100, pool());
+  PartitionIdGenerator idGenerator(ROW({VARCHAR()}), {0}, 100, pool(), true);
```

Comment:
Can we add a test where value of new flag is 'false'?

Reply:
@mbasmanova Yes. Added a new test the test `false `value.